metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -59,10 +61,22 @@ public ParserBolt withMessageFilter(MessageFilter<JSONObject> filter) {
   public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
     super.prepare(stormConf, context, collector);
     this.collector = collector;
+    if(getSensorParserConfig() == null) {
+      filter = new GenericMessageFilter(null);
+    }
+    else if(filter == null) {
+      filter = Filters.get(getSensorParserConfig().getFilterClassName()
+              , getSensorParserConfig().getParserConfig()
+      );
+    }
     parser.init();
     writer.init();
   }
 
+  protected SensorParserConfig getSensorParserConfig() {
```

Comment:
When I went to change it, I realized why I put it in the ParserBolt rather than the ConfiguredParserBolt.  The sensorType is part of the ParserBolt rather than the ConfiguredParserBolt.  I am going to make them both part of ConfiguredParserBolt.